vector/src/main/java/im/vector/app/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -121,30 +122,31 @@ import org.matrix.android.sdk.internal.database.lightweight.LightweightSettingsS
 import javax.inject.Inject
 
 class MessageItemFactory @Inject constructor(
```

Comment:
it's very tricky to know what changed in this file due to the formatting :cry:

Reply:
Sorry, I will try to revert to only the changes I have done. I guess this class will be reformatted in the future with new coding styles.

Reply:
if you're able to point out the changes that would be enough for me (rather than having to revert)

Reply:
Too late 😄 Now you can see clearly the changes.